install/kubernetes/helm/istio/charts/grafana/dashboards/pilot-dashboard.json

```diff
@@ -713,42 +712,96 @@
       "steppedLine": false,
       "targets": [
         {
-          "expr": "label_replace(sum(pilot_xds_cds_reject{job=\"pilot\"}) by (node, err), \"node\", \"$1\", \"node\", \".*~.*~(.*)~.*\")",
+          "expr": "sum(rate(pilot_xds_cds_reject{job=\"pilot\"}[1m]))",
           "format": "time_series",
           "hide": false,
           "intervalFactor": 1,
-          "legendFormat": "Rejected CDS Configs - {{ node }}: {{ err }}",
+          "legendFormat": "Rejected CDS Configs",
           "refId": "C"
         },
         {
-          "expr": "pilot_xds_eds_reject{job=\"pilot\"}",
+          "expr": "sum(rate(pilot_xds_eds_reject{job=\"pilot\"}[1m]))",
```

Comment:
Looks like this is also in ADS monitoring section? Also some have `hide: true`, is that intended?